server/src/files.ts

```diff
@@ -209,12 +209,23 @@ export function resolveModulePath(workspaceRoot: string, moduleName: string, nod
 			nodePath = path.join(workspaceRoot, nodePath);
 		}
 		return resolve(moduleName, nodePath, nodePath, tracer).then((value) => {
+			if (tracer) {
+				tracer(`Resolved module path is ${value}`);
+			}
 			if (value.indexOf(path.normalize(nodePath)) === 0) {
```

Comment:
I think the issue lies with this `indexOf` check. It could be something as simple as `c:/` vs `C:/` in the path name.

Between this trace statement and the one in the `else` case, it should be hopefully be pretty clear what is going wrong.